app_dart/lib/src/request_handling/static_file_handler.dart

```diff
@@ -0,0 +1,145 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:appengine/appengine.dart';
+import 'package:meta/meta.dart';
+import 'package:mime/mime.dart';
+
+import 'exceptions.dart';
+
+/// A class based on [RequestHandler] for serving static files.
+@immutable
+class StaticFileHandler {
```

Comment:
This should extend [`RequestHandler<Body>`](https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/request_handling/request_handler.dart) and override `get()`.  It can probably return `Body.forStream()` on the stream of bytes you get when you open the file.

Reply:
Updated. Thank you for the explanation